src/astt_gui/templates/index.html

```diff
@@ -226,26 +267,31 @@ <h4>Functional State</h4>
                         <div class="container">
                           <div class="row">
                             <div class="col">
-                              <button onclick="openInitModal('')" class="btn btn-primary"><i class='bx bxs-right-arrow'></i> Initialize System</button>
+                              <button onclick="openInitModal('')" class="btn btn-primary" title="This button initializes the system"> <i class='bx bxs-right-arrow'></i> Initialize System</button>
                             </div>
                             <div class="col">
-                              <button class="btn btn-primary"><span class="bi bi-qr-code"></span> Scan Source/s</button>
+                              <button class="btn btn-primary" title="This button is used when we want to scan for sources i.e the sun also the stars in the evening"><span class="bi bi-qr-code"></span> Scan Source/s</button>
+                              <!--<div class="btn-container">
+                                <button class="btn btn-primary"><span class="bi bi-qr-code"></span> Scan Source/s</button>
+                                <span class="tooltip">This is where we display sources</span>
+                              </div> -->
                             </div>
                             <div class="col">
-                              <button class="btn btn-primary"  data-bs-toggle="modal" data-bs-target="#modeModal"><span class='bi bi-arrows-collapse'></span> Antenna Modes</button>
+                              <button class="btn btn-primary" title="This button is used to switch between different antenna modes namely, Idle, Stow and Point."  data-bs-toggle="modal" data-bs-target="#modeModal"><span class='bi bi-arrows-collapse'></span> Antenna Modes</button>
```

Comment:
This looks good and I tested it locally however, we can extend the statements to give more information about what the button does.